### PR TITLE
feat: add launchpad download section

### PR DESCRIPTION
**Pull request by CjS77**
Adds section for Launchpad downloads.

TODO:
 
 * [x]  Js and YML needs to be updated to point to the launchpad binaries, with correct filters (when they exist)

### CODE REVIEW

File: _data/launchpad_downloads.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  title: Minotari Launchpad for Ubuntu
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: linuxDL
+  checksumDivID: linuxCSID
+  version: Ubuntu 18.04 and higher
+  source: https://github.com/Minotari-project/Minotari-launchpad
+  background-image: /assets/img/downloads/launchpad-ubuntu.mp4
+  video-poster: /assets/img/downloads/ubuntu.png
+  tutorial-video: /dibbler-install-guides/
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  tutorial-video: /launchpad-guide/
```

File: _data/launchpad_downloads.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  title: Minotari Launchpad for Mac
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: osxDL
+  checksumDivID: osxCSID
+  version: MacOS 10.15.0 (Catalina) and higher
+  source: https://github.com/Minotari-project/Minotari
+  background-image: /assets/img/downloads/launchpad-mac.mp4
+  video-poster: /assets/img/downloads/mac.png
+  tutorial-video: /dibbler-install-guides/
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  tutorial-video: /launchpad-guide/
```

File: _data/launchpad_downloads.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  title: Minotari Launchpad for Windows
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: windowsDL
+  checksumDivID: windowsCSID
+  version: Windows 10 and higher
+  source: https://github.com/Minotari-project/Minotari
+  background-image: /assets/img/downloads/launchpad-windows.mp4
+  video-poster: /assets/img/downloads/windows.png
+  tutorial-video: /dibbler-install-guides/
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  tutorial-video: /launchpad-guide/
```

File: _data/launchpad_downloads.yml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  archOptions: [{ value: x64, label: x64 }]
+  networkOptions: [{ value: nextnet, label: Nextnet }]
+- type: Libraries
+  elementID: support
+  binaryDivID: libWalletBinID
+  elementTabButtonID: support-btn
+  title: Minotari Support Libraries
+  subtitle: Additional miscellaneous binaries and files to enable developers.
+  downloadBtnID: null
+  tutorial-video: null
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  tutorial-video: /launchpad-guide/
```

File: _data/launchpad_downloads.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  archDropdownID: linuxArchID
+  networkDropdownID: linuxNetworkID
+  elementTabButtonID: ubuntu-btn
+  title: Minotari Launchpad for Ubuntu
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: linuxDL
+  checksumDivID: linuxCSID
+  version: Ubuntu 18.04 and higher
+  source: https://github.com/Minotari-project/Minotari-launchpad
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  source: https://github.com/tari-project/tari-launchpad
```

File: _data/launchpad_downloads.yml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  archDropdownID: osxArchID
+  networkDropdownID: osxNetworkID
+  elementTabButtonID: mac-btn
+  title: Minotari Launchpad for Mac
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: osxDL
+  checksumDivID: osxCSID
+  version: MacOS 10.15.0 (Catalina) and higher
+  source: https://github.com/Minotari-project/Minotari
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  source: https://github.com/tari-project/tari-launchpad
```

File: _data/launchpad_downloads.yml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+  archDropdownID: windowsArchID
+  networkDropdownID: windowsNetworkID
+  elementTabButtonID: windows-btn
+  title: Minotari Launchpad for Windows
+  subtitle: Download the binary, click through to install. Then you'll be
+    automatically connected to the Minotari blockchain.
+  downloadBtnID: windowsDL
+  checksumDivID: windowsCSID
+  version: Windows 10 and higher
+  source: https://github.com/Minotari-project/Minotari
```
Comment (CjS77) [inline _data/launchpad_downloads.yml]:
```suggestion
  source: https://github.com/tari-project/tari-launchpad
```

**Review by CjS77 — COMMENTED** [review]
Fixing links

**Comment by CjS77** [comment]
ACK

**Review by SWvheerden — APPROVED** [review]

**Comment by SWvheerden** [comment]
ACK

**Pull request merged**